### ISSUE TITLE
New dsl, step 4/3? split and move Style from Dom to Dom_html and Dom_svg

### ISSUE THREAD

**Issue by glennsl** [ml-in-barcelona/jsoo-react#137]
Seems it would be more consistent to divide the style rules the same way we do with tags and props. What do you think?

**Comment by jchavarri** [comment]
Yeah I agree. It could be step 3/3, as it's now unclear if we'll have per-element prop types 😬
